### ISSUE TITLE
Write custom adapter for sklearn gradient computation

### ISSUE THREAD

**Issue by maurapintor** [pralab/secml-torch#1]
Already implemented in SecML, using the `nn.Module` wrapper.
Only SVM for now.
